client/gutenberg/editor/controller.js

```diff
@@ -58,6 +61,27 @@ export const post = ( context, next ) => {
 			return;
 		}
 
+		const calypsoifyGutenberg =
+			isEnabled( 'gutenberg' ) &&
+			isEnabled( 'gutenberg/opt-in' ) &&
+			isEnabled( 'calypsoify/gutenberg' ) &&
```

Comment:
We can probably trim this down to one feature flag and also default that off in envs until we're ready.